### PR TITLE
Content tweaks to onboarding journey.

### PR DESCRIPTION
**Pull request by frankieroberto**
Thanks to @Deepatee for reviewing.

### CODE REVIEW

File: app/views/regions/v1/add-organisation.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
       <h1 class="nhsuk-heading-xl">Invite an organisation</h1>
 
-      <p>Possibly some guidance here if needed?</p>
+      <p>The organisation will be invited to start setting up their accounts for the service, but will not be able to start recording vaccinations until you activate them later.</p>
```
Comment (frankieroberto) [inline app/views/regions/v1/add-organisation.html:15]: @Deepatee how about some kind of content like this to try and explain that they're inviting organisations to set up their accounts ahead of getting the go-ahead to start actually using it? 🤔

Comment (Deepatee) [reply app/views/regions/v1/add-organisation.html:15]: Hi @frankieroberto, I'm just checking: Does this appear on the screen where they search for their NHS trust by ODS code or name? Do you know if it's the region that gives them the go-ahead to record?

Comment (frankieroberto) [reply app/views/regions/v1/add-organisation.html:15]: @Deepatee yep, this is the screen where they search for the NHS trust in their region.

My understanding is that it's the regions who give the go-ahead to record vaccinations (after signing contracts and doing assurance). We’ll check all this in the research though!

Comment (Deepatee) [reply app/views/regions/v1/add-organisation.html:15]: Ignore me, could not find and see the new line of content (15). Found it. How about...

Invite an organisation to set up an account first. Later, you can activate it to allow them to record vaccinations.

Comment (frankieroberto) [reply app/views/regions/v1/add-organisation.html]: Nice, thanks!

**Pull request merged**